src/lib/components/WelcomeModal/WelcomeModal.svelte

```diff
@@ -3,10 +3,14 @@
   import { writable } from 'svelte/store';
   import { onMount, onDestroy } from 'svelte';
   import { welcomeModalStore } from './store';
+  import { goto } from '$app/navigation';
 
+  const orgSlug = orgData.siteName;
   const closeModal = () => {
     $welcomeModalStore.open = false;
+    goto('/org/${orgSlug}/courses?create=true');
   };
```

Comment:
Hi, thanks a lot for creating this PR.

This code should lead to an error because `orgData` is undefined. Did you run and test the code locally?

To get the url for the org slug there is a store value called `currentOrgPath`. You can import it like

```ts
import { currentOrgPath } from '$lib/utils/store/org';

// then use it this way
goto($currentOrgPath + '/courses?create=true')
```

This way you should have the orgSlug already added dynamically.